frontend/packages/dev-console/integration-tests/testData/git-import-flow.data.ts

```diff
@@ -13,3 +16,57 @@ export const testData = {
   scalingLimit: 2,
   label: 'app=devlelopment',
 };
+
+export const devStrategyInput = {
```

Comment:
Should be called devfile

```suggestion
export const devfileStrategyInput = {
```